### ISSUE TITLE
icon-sdk-js randomly throwing Unexpected token < in JSON at position 0

### ISSUE THREAD

**Issue by 0xrpj** [icon-project/icon-sdk-js#72]
Bug: This is a very weird bug that I have faced in production. Have occurred twice in the last month. I couldn't exactly pinpoint the place which caused this because occurs rarely in the codebase.

I suspect that this might have been triggered through a getter call. Please look into it.

![image](https://github.com/icon-project/icon-sdk-js/assets/56220537/e360a466-87ac-4df2-a8f4-93628c623db3)

Here is the implementation:

```
  public async invokeCallBuilder(score: string, method: string, params: any): Promise<unknown> {
    try {
      const call = new CallBuilder().to(score).method(method).params(params).build();
      const response = await this.iconService.call(call).execute();
      return response;
    } catch (error) {
      throw new Error(error);
    }
  }
```

**Comment by han-so1omon** [comment]
@inwonkim Do you have any comments on this?

**Comment by inwonkim** [comment]
Need more information.
Can you inform what data is passed to `params`?

**Comment by inwonkim** [comment]
@0xrpj Can I get the endpoint, SCORE address information?
The response value seems strange.

**Comment by mksong-icon** [comment]
According to the discord comments, it uses https://ctz.solidwallet.io/api/v3 for the endpoint.
In node implementation, it doesn't send any HTML response. So, I think that it may come from HTTPS reverse proxy which is applied to the citizens. Nowadays, some nodes crash from JSON RPC Bug(which is fixed in v1.3.7).
So, it may cause HTML response from the reverse proxy and an exception in parsing JSON.
